src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -3728,24 +3802,6 @@ def test_rc_invalid_depth():  # pylint: disable=invalid-name
             "the supplied depth is less than 1") in str(excinfo)
 
 
-def test_rc_invalid_depth_continuous():  # pylint: disable=invalid-name
```

Comment:
As we now support redundant computation the error has changed to be that the depth must increase, rather than be greater than 1. This test has been replaced with test_rc_colour_no_loop_decrease().